morpho/processors/IO/IOROOTProcessor.py

```diff
@@ -64,16 +64,17 @@ def Reader(self):
                 import ROOT
             except ImportError:
                 logger.warning("Failed importing ROOT")
-                pass
```

Comment:
The logic below is completely different from what it was before... And indeed it doesn't seem to work.
I will push a new commit that is supposed to fix the logic.